pages/add.js

```diff
@@ -42,11 +40,7 @@ export default function Add({ searchedPlayer, players, setPlayers }) {
 }
 
 const StyledLink = styled.a`
-  position: relative;
+  position: fixed;
+  top: 50%;
```

Comment:
to have this link fixed (at 50%) can lead to unexpected behaviour, e.g. if the viewport-height is small it will appear above the other stuff. Why don't append it with a top-margin?